web/degausser.js

```diff
@@ -2,11 +2,16 @@ globalThis.degausser = input => {
   var div = document.createElement("div");
   div.innerHTML = input;
   document.body.appendChild(div);
-  Array.from(div.querySelectorAll("a, img, area, input")).forEach(el => {
+  Array.from(div.querySelectorAll("a, input")).forEach(el => {
     if (el.childNodes.length === 0) {
       el.outerHTML = el.alt || "";
     }
   });
+  Array.from(div.querySelectorAll("img, area, image")).forEach(el => {
+    if (el.childNodes.length === 0) {
+      el.outerHTML = el.alt || String.fromCharCode(31).repeat(100);
```

Comment:
Let's avoid magic numbers and label this semantically by defining it as a named constant

Reply:
Done.